CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+# Contributing to TikTock
+
+Thank you for considering contributing to TikTock. We appreciate your interest in making TikTock better!
+
+## How to Contribute
+
+To contribute to TikTock, please follow these steps:
+
+1. Fork the repository to your GitHub account.
+2. Clone the forked repository to your local machine.
+3. Create a new branch for your changes.
+
```

Comment:
Maybe add something like "4. Make a pull request once you're satisfied with the changes"?